third_party/terraform/resources/resource_iam_audit_config.go

```diff
@@ -39,35 +39,46 @@ var iamAuditConfigSchema = map[string]*schema.Schema{
 }
 
 func ResourceIamAuditConfig(parentSpecificSchema map[string]*schema.Schema, newUpdaterFunc newResourceIamUpdaterFunc, resourceIdParser resourceIdParserFunc) *schema.Resource {
+	return ResourceIamAuditConfigWithBatching(parentSpecificSchema, newUpdaterFunc, resourceIdParser, IamBatchingDisabled)
+}
+
+func ResourceIamAuditConfigWithBatching(parentSpecificSchema map[string]*schema.Schema, newUpdaterFunc newResourceIamUpdaterFunc, resourceIdParser resourceIdParserFunc, enableBatching bool) *schema.Resource {
```

Comment:
Just curious, is `enableBatching` temporary while we roll out batching to all IAM-based resources or do you expect it will exist permanently?

Reply:
temporary, I'm hoping to roll out to all but that seemed like a YOLO change to do immediately (tbh project is also the one that most people use probably so it's already a big change)